plugins/source/aws/client/testing.go

```diff
@@ -150,3 +153,36 @@ func validateNoEmptyColumnsExcept(t *testing.T, tables schema.Tables, messages m
 		}
 	}
 }
+
+func validateTagData(t *testing.T, tables schema.Tables, messages message.SyncMessages, except map[string][]string) {
+	tablesWithIssues := []string{}
+	for _, table := range tables.FlattenTables() {
+		if table.Column("tags") == nil {
+			continue
+		}
+
+		records := messages.GetInserts().GetRecordsForTable(table)
+		for _, resource := range records {
+			for colIndex, arr := range resource.Columns() {
+				if table.Columns[colIndex].Name != "tags" {
```

Comment:
get index by name & use only that?
https://pkg.go.dev/github.com/cloudquery/plugin-sdk/v4@v4.38.0/schema#ColumnList.Index